draft-ietf-keytrans-protocol.md

```diff
@@ -1190,23 +1441,54 @@ struct {
 In turn, the Transparency Log responds with a SearchResponse structure:
 
 ~~~ tls-presentation
+struct {
+  opaque proof<VRF.Np>;
+  opaque commitment<Hash.Nh>;
+} BinaryLadderStep;
+
 struct {
   FullTreeHead full_tree_head;
 
-  SearchProof search;
-  opaque opening<16>;
+  optional<uint32> version;
+  BinaryLadderStep binary_ladder<0..2^8-1>;
```

Comment:
I don't understand why there is a single binary ladder here. Don't we need multiple (for each node on the path in the implicit binary search tree to the leaf)? And aren't those provided in `search`?